src/elk-monitoring/pagopa/ingest-pipeline.json

```diff
@@ -0,0 +1,11 @@
+{
+  "processors": [
+    {
+      "json": {
+        "field": "message",
+        "target_field": "app",
```

Comment:
```suggestion
        "add_to_root": true,
```

Reply:
This way the content of message kubernetes ingested log will replace the original message. All ECS message fields will be then correctly set into ingested log making logs visible inside APM logs section and so on.
I modified pipeline manually on dev environment. Here you can see an example of ingested log that are also visible inside apm logs service log section that are ingested by filebeet (apm log sending feature is turned off https://kibana.dev.platform.pagopa.it/kibana/app/apm/services/pagopa-ecommerce-transactions-service/logs?comparisonEnabled=true&environment=ENVIRONMENT_ALL&kuery=&latencyAggregationType=avg&offset=1d&rangeFrom=now-15h&rangeTo=now&serviceGroup=&transactionType=request)